edtf/templates/edtf/edtf_form.html

```diff
@@ -70,7 +74,8 @@ <h1>
 <h4>Parameters</h4>
 <b>date</b>: <i>Required</i> - the text respresentation of the date you want to verify.<br>
 <b>callback</b>: <i>Optional</i> - the text represenation of the jsonp callback wrapper.<br>
-<b>level</b>: <i>Optional</i> - the level at which you would like the given date verified against. If none is given, the service assumes you are verifying against the full EDTF spec.
+<b>levelFeature</b>: <i>Optional</i> - the level at which you would like the given date should be a feature of. If none is given, the service assumes you are verifying against the full EDTF spec.<br>
```

Comment:
I think this wording needs to be tweaked sightly. Perhaps something like: "...the level at which you would like the given date checked for being a feature of..."

Reply:
Also, what do you think about removing "If none is given, the service assumes you are verifying against the full EDTF spec." from the `levelFeature` and `levelConfoms` lines here, since it only makes sense if both `levelFeature` and `levelConforms` are left off the request?

Reply:
I think it's ok to let it be here cos, it mentions if the optional arguments `levelFeature` and `levelConforms` are not given, the date will then be validated against full edtf spec. I can have it removed if you think otherwise.

Reply:
The way I read it, if you don't have `levelFeature`, whether or not you have `levelConforms`, it would be validating against the full spec. Maybe "If none" should be changed to "If neither `levelFeature` nor `levelConforms`" and, in the next line, "If neither `levelConforms` nor `levelFeature`" Or perhaps the "If none" sentences should be removed and a "Note:" added at the end of the Parameters section that indicated `levelFeature` and `levelConforms` are mutually exclusive, and if neither is given, the `date` will be validated against the full EDTF specification.